modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,166 @@
+
+If you use a version-based channel, you can always upgrade to the latest version in the channel, or head, for example, from 1.29.0 to 1.29.2 in the `stable-1.29` channel.
+
+Additionally, from the head of the channel, you can upgrade to the next `x.y` release. For example, if 1.29.2 is the head in the `stable-1.29` channel, you can upgrade from 1.29.2 to 1.30. Such cross-channel updates are not done automatically, and the administrator needs to switch the channel manually by updating the subscription.
+
+== Upgrade examples
+
+=== Scenario 1
+
+In this scenario:
+
+* the channel is `stable-1.28`
+* you are switching to `stable`
+* the currently installed version is 1.28.0
+* 1.29.0 was released earlier than 1.28.1
+* 1.30.0 is the head of `stable`
+
+The upgrade path from 1.28.0 on `stable-1.28` to 1.29.0 on `stable` in this scenario is 1.28.0 to 1.28.1 to 1.29.0.
+
+[source]
+----
+                                            
+     stable                    stable-1.28  
++--------------+            +--------------+
+|              |            |              |
+|  +--------+  |            |  +--------+  |
+|  | 1.28.0 |  |            |  | 1.28.0 |  |
+|  +--------+  |            |  +--------+  |
+|              |            |       |      |
+|              |            |       |      |
+|  +--------+  |            |       |      |
+|  | 1.29.0 |<--------      |       v      |
+|  +--------+  |     |      |  +--------+  |
+|              |     +---------| 1.28.1 |  |
+|              |            |  +--------+  |
+|  +--------+  |            |              |
+|  | 1.30.0 |  |            |              |
+|  +--------+  |            |              |
+|              |            |              |
++--------------+            +--------------+
+----
+
+=== Scenario 2
+
+In this scenario:
+
+* the channel is `stable-1.29`
+* the currently installed version is 1.29.0
+* 1.29.1 was released to both `stable-1.29` and `stable` before `1.30.0` was released to `stable`
+
+The upgrade path from 1.29.0 on `stable-1.29` to 1.30.0 on `stable` in this scenario is 1.29.0 to 1.29.1 to 1.30.0.
+
+[source]
+----
+                                           
+     stable                   stable-1.29  
++--------------+           +--------------+
+|              |           |              |
+|  +--------+  |           |  +--------+  |
+|  | 1.29.0 |  |           |  | 1.29.0 |  |
+|  +--------+  |           |  +--------+  |
+|              |           |       |      |
+|              |           |       v      |
+|  +--------+  |           |  +--------+  |
+|  | 1.29.1 |  |           |  | 1.29.1 |  |
+|  +--------+  |           |  +--------+  |
+|              |           |       |      |
+|              |           |       |      |
+|  +--------+  |           |       |      |
+|  | 1.30.0 |<---------------------+      |
+|  +--------+  |           |              |
+|              |           |              |
++--------------+           +--------------+
+----
```

Comment:
Same feedback regarding my comment above about requesting a diagram and excluding this from the PR in the meantime.